### ISSUE TITLE
Useful tutorial

### ISSUE THREAD

**Issue by qzhou711** [Zarkopafilis/awesome-dronecraft#3]
To further expand the audience, I can provide the Chinese version ：）

**Comment by ntakouris** [comment]
Yes please! Do you mind opening up a branch here (as a PR -- and then we can link the chinese version on the original readme) ?
